src/makeCalculator.js

```diff
@@ -4,7 +4,28 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
```

Comment:
[NAMING] - The name `calculator` for the object is not appropriate in this context. It would be more appropriate to call it `calc` or `calculatorObject`.